topics/icrc_28_trusted_origins.md

```diff
@@ -116,46 +125,44 @@ sequenceDiagram
     participant RP as Relying Party
     participant S as Chain Connected Signer Component
     participant CS as Cold Signer Component
-    participant B as Boundary Node
-    participant U as User
+    participant C as Target Canister
     Note over RP, S: Interactions follow ICRC-34 standard
-    RP ->> CS: Request global delegation
+    RP ->> S: Request global delegation
     loop For every target canister
-        CS ->> B: Metadata read state request
-        B ->> CS: Metadata response
+        S ->> C: Get trusted origins
+        C ->> S: List of trusted origins
     end
-    S ->> CS: - Relying party origin<br>- Delegation request<br>- Metadata responses
+    S ->> CS: - Relying party origin<br>- Delegation request<br>- Trusted origins responses
```

Comment:
Given that the certificate only contains the `requestId` and not the initial request, we also need to have the trusted origins requests available in the cold component. Otherwise the cold signer cannot verify whether these responses were generated a) by the correct canisters b) for the correct method_call.